src/main/java/com/google/sps/data/EventStorage.java

```diff
@@ -47,22 +76,31 @@ public static void addEvent(Event event) {
     eventEntity.setProperty("declined-users", event.getDeclinedIDs());
 
     // Store Entities to datastore.
-    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
-    datastore.put(eventEntity);
+    DatastoreServiceFactory.getDatastoreService().put(eventEntity);
   }
 
   public static void editEvent(Event event) {
-    // TODO: Edit event in datastore.
+    deleteEvent(event.getID());
+    addEvent(event);
```

Comment:
Is the Datastore.put method not updating the event? So you don't need to delete the event. If this is the case, maybe rename the addEvent method to addOrUpdateEvent.